smartapps/konnected-io/konnected-security-connect.src/konnected-security-connect.groovy

```diff
@@ -127,11 +126,7 @@ def pageDiscovery() {
   //create accessToken
   if(!state.accessToken) { createAccessToken() }
 
-  //This is a workaround to prevent page to refresh too fast.
-  if(!state.pageConfigurationRefresh) { state.pageConfigurationRefresh = 2 }
-
-  dynamicPage(name: "pageDiscovery", nextPage: "pageConfiguration", refreshInterval: state.pageConfigurationRefresh) {
-    state.pageConfigurationRefresh =  state.pageConfigurationRefresh + 3
```

Comment:
https://github.com/SmartThingsCommunity/SmartThingsPublic/issues/2082
https://community.smartthings.com/t/bug-issue-with-dynamic-page/89625

once it hits initialize, it will set it back to 2 again.